pkgs/by-name/su/sudo-rs/package.nix

```
@@ -78,13 +82,16 @@ rustPlatform.buildRustPackage rec {
   meta = with lib; {
```

Since you're making a cleanup, let's remove this too.

```suggestion
  meta = {
```